dockers/Screenshotter/screenshotter.js

```diff
@@ -231,7 +271,20 @@ function takeScreenshot(key) {
             key += "_alt";
         }
         var file = path.join(dstDir, key + "-" + opts.browser + ".png");
-        fs.writeFile(file, img.buf, check);
+        var deferred = new selenium.promise.Deferred();
+        fs.writeFile(file, img.buf, function(err) {
+            if (err) {
+                deferred.reject(err);
+            } else {
+                deferred.fulfill();
+            }
+        });
+        return deferred.promise;
+    }).then(function() {
         console.log(key);
+        if (--countdown === 0) {
+            // devServer.close(cb) will take too long.
+            process.exit(0);
```

Comment:
Feels like forever, but realistically it's probably the one minute commonly configured as [`TIME_WAIT`](https://en.wikipedia.org/wiki/Transmission_Control_Protocol#Protocol_operation) for a TCP connection.